iis-workflows/iis-workflows-ingest-pmc/src/main/java/eu/dnetlib/iis/workflows/ingest/pmc/metadata/MetadataImporter.java

```diff
@@ -46,15 +85,29 @@ protected void map(AvroKey<DocumentText> key, NullWritable value, Context contex
 				saxParser.parse(new InputSource(
 						new StringReader(nlm.getText().toString())), 
 						pmcXmlHandler);
-                context.write(new AvroKey<ExtractedDocumentMetadata>(output.build()), 
-                		NullWritable.get());
+				mos.write(namedOutputMeta, 
+						new AvroKey<ExtractedDocumentMetadata>(output.build()));
         	 } catch (ParserConfigurationException e) {
-             	throw new IOException("Text extraction failed for id " + nlm.getId() + 
-             			" and text: " + nlm.getText(), e);
+        		 Map<CharSequence, CharSequence> auditSupplementaryData = new HashMap<CharSequence, CharSequence>();
+        			auditSupplementaryData.put(FAULT_TEXT, nlm.getText());
```

Comment:
Do we know all possible scenarios of `auditSupplementaryData` usage? I don't know this is why I didn't mention about possible values. So far I've been using `text` and `processing_time` each one in different context and processing module. We could describe it e.g. in schema but this will beg for complex explanation and implicitly require schema comment updates whenever new audit data type is used. I am not sure schema is the right place for this.